internal/accounts/accounts.go

```diff
@@ -86,6 +86,13 @@ func (r *accountResult) String() string {
 	var b bytes.Buffer
 	writer := util.CreateTabWriter(&b)
 
+	if r.Address.IsValid(flow.Testnet) {
+		_, _ = fmt.Fprintf(writer, fmt.Sprintf(
+			"If you would like to fund the account with 1000 FLOW tokens for testing,"+
+				" visit %s \n\n",
+			fmt.Sprintf("https://testnet-faucet.onflow.org/fund-account?address=%s", r.Address.String())))
```

Comment:
`Fprintf` already formats, so no need for extra formats inside:

```suggestion
		_, _ = fmt.Fprintf(
			writer,
			"If you would like to fund the account with 1000 FLOW tokens for testing,"+
				" visit https://testnet-faucet.onflow.org/fund-account?address=%s\n\n",
		 	r.Address.String(),
		)
```

Reply:
updated and thank you!